test/lib/clients/security_proposal_test.rb

```diff
@@ -136,15 +136,15 @@
       end
     end
 
-    context "when selinux is configurable" do
-      it "contains in proposal selinux configuration" do
-        allow(proposal_settings.selinux_config).to receive(:configurable?)
+    context "when LSM is configurable" do
+      it "contains in proposal LSM configuration" do
```

Comment:
We're testing the proposal, let's simplify the test description (which comes from the previous changes, I know)

```suggestion
      it "contains the LSM configuration" do
```